demo/kserve/create-minio.md

```diff
@@ -1,27 +1,44 @@
-# Create your own Minio image
+# Create your own MinIO image
 
-If you would to create your own minio image with a desired model, first fork https://github.com/kserve/modelmesh-minio-examples.
+You can create your own MinIO image with a model.
 
-1. Clone the repo (update `<YOUR-USERNAME>`) and navigate to the correct directory.
-~~~
-git clone https://github.com/<YOUR-USERNAME>/modelmesh-minio-examples.git
-cd modelmesh-minio-examples
-~~~
+**Prerequisite**
 
-2. Create your model directory (update `<yourmodeldirectory>`) 
-~~~
-mkdir <yourmodeldirectory> && cd <yourmodeldirectory>
-~~~
-and place your model there.
+* You created a fork of the https://github.com/kserve/modelmesh-minio-examples GitHub repository.
 
+* You created a model.
 
-3. Edit the Dockerfile (located in the main directory) to add lines for your directory following the pattern [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L36) and [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L59).
+* You have a GitHib account.
 
-4. Build the image:
-~~~
-podman build -t quay.io/`<YOUR_QUAY_USERNAME>`/modelmesh-minio-examples:`<YOUR_TAG>` .
-~~~
-5. Push the image to quay:
-~~~
-podman push quay.io/`<YOUR_QUAY_USERNAME>`/modelmesh-minio-examples:`<YOUR_TAG>`
-~~~
\ No newline at end of file
+* You have a Quay account.
+
+**Procedure**
+
+1. Clone the repo (replace `<YOUR-USERNAME>` with your GitHub username).
+   ~~~
+   git clone https://github.com/<YOUR-USERNAME>/modelmesh-minio-examples.git
+   ~~~
+
+2. Navigate to the `modelmesh-minio-examples` directory.
+   ~~~
+   cd modelmesh-minio-examples
+   ~~~
+
+3. Create a model directory (replace `<yourmodeldirectory>` with the name of a directory for your model).
+   ~~~
+   mkdir <yourmodeldirectory>
+   ~~~
+
+4. Copy your model into the `modelmesh-minio-examples/<yourmodeldirectory>` directory .
+
+5. Edit your local `main/Dockerfile` and add lines for your model directory by following the pattern [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L36) and [here](https://github.com/kserve/modelmesh-minio-examples/blob/main/Dockerfile#L59).
+
+6. Build the image:
+   ~~~
+   podman build -t quay.io/`<YOUR_QUAY_USERNAME>`/modelmesh-minio-examples:`<YOUR_TAG>` .
```

Comment:
what is the value of <YOUR-TAG>? How does the user know its value?

Reply:
It's up to the user to set a tag. It could be a `v1` or `model-name` or even `latest` (for example: https://quay.io/repository/opendatahub/modelmesh-minio-examples?tab=tags&tag=latest) . Or maybe we can set it as `v1` to avoid confusion.